### PR TITLE
Correct suggested regularization values

### PR DESCRIPTION
**Pull request by Hydrotoast**
If default regularization is L2, the regularization constant should be less than 1. 

For the case of gradient descent: `new_param = old_param - eta * ( lambda * old_param + gradient)` where `eta` is the learning rate. Suppose `eta = 1` for simplicity. If `lambda = 1`, this would annihilate the parameter and only learn the gradient at each step. If `lambda > 1`, this would oscillate.

### CODE REVIEW

**Comment by fastier-li** [comment]
@Hydrotoast - `eta` in our implementation is actually adaptive, rather than fixed. It has a decay schedule to try and avoid oscillations.

**Comment by fastier-li** [comment]
At this point, we are not planning on merging this PR, as the training rate is actually adaptive, so we shouldn't get into oscillations.

**Comment by fastier-li** [comment]
Closing after talking with Ankan and other scientists on our team.
